MrtCore/winui_desktop_packaged_app/ClassLibrary/Class.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text;
+using Microsoft.ApplicationModel.Resources;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace ClassLibrary
+{
+    public class Class
+    {
+        public string GetClassLibrarySampleString()
+        {
+            var resourceManager = new ResourceManager();
+            return resourceManager.MainResourceMap.GetValue("ClassLibrary/ClassLibraryResources/ClassLibrarySampleString").ValueAsString;

```

Comment:
>ClassLibrary/ClassLibraryResources/ClassLibrarySampleString [](http://example.com/codeflow?start=61&length=59)

is this the path in the PRI?

Reply:
Yup:

```
		<ResourceMapSubtree name="ClassLibrary">
			<ResourceMapSubtree name="ClassLibraryResources">
				<NamedResource name="ClassLibrarySampleString" uri="ms-resource://323858f2-34e1-48a5-8244-5c1bc0c1f889/ClassLibrary/ClassLibraryResources/ClassLibrarySampleString">
					<Candidate type="String">
						<Value>English Sample from Class Library</Value>
					</Candidate>
				</NamedResource>
			</ResourceMapSubtree>
		</ResourceMapSubtree>
```

Reply:
is the format "[project name]/[resw filename]/[string name]"?